runelite-client/src/main/java/net/runelite/client/plugins/profiles/ProfilesPanel.java

```diff
@@ -0,0 +1,222 @@
+	private final JPasswordField txtAccountLogin = new JPasswordField(ACCOUNT_USERNAME);
+	private final List<ProfilePanel> profiles = new ArrayList<>();
+	private GridBagConstraints c;
+
+	@Inject
+	public ProfilesPanel(Client client, ProfilesConfig config)
+	{
+		super();
+		this.client = client;
+		profilesConfig = config;
+
+		setBorder(new EmptyBorder(18, 10, 0, 10));
+		setBackground(ColorScheme.DARK_GRAY_COLOR);
+		setLayout(new GridBagLayout());
+
+		c = new GridBagConstraints();
+		c.fill = GridBagConstraints.HORIZONTAL;
+		c.gridx = 0;
+		c.gridy = 0;
+		c.weightx = 1;
+		c.weighty = 0;
+		c.insets = new Insets(0, 0, 3, 0);
+
+		txtAccountLabel.setPreferredSize(PREFERRED_SIZE);
+		txtAccountLabel.setForeground(ColorScheme.MEDIUM_GRAY_COLOR);
+		txtAccountLabel.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+		txtAccountLabel.setMinimumSize(MINIMUM_SIZE);
+		txtAccountLabel.addFocusListener(new FocusListener()
+		{
+			@Override
+			public void focusGained(FocusEvent e)
+			{
+				if (txtAccountLabel.getText().equals(ACCOUNT_LABEL))
+				{
+					txtAccountLabel.setText("");
+					txtAccountLabel.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
+				}
+			}
+
+			@Override
+			public void focusLost(FocusEvent e)
+			{
+				if (txtAccountLabel.getText().isEmpty())
+				{
+					txtAccountLabel.setForeground(ColorScheme.MEDIUM_GRAY_COLOR);
+					txtAccountLabel.setText(ACCOUNT_LABEL);
+				}
+			}
+		});
+
+		add(txtAccountLabel, c);
+		c.gridy++;
+
+		// Do not hide username characters until they focus or if in streamer mode
+		txtAccountLogin.setEchoChar((char) 0);
+		txtAccountLogin.setPreferredSize(PREFERRED_SIZE);
+		txtAccountLogin.setForeground(ColorScheme.MEDIUM_GRAY_COLOR);
+		txtAccountLogin.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+		txtAccountLogin.setMinimumSize(MINIMUM_SIZE);
+		txtAccountLogin.addFocusListener(new FocusListener()
+		{
+			@Override
+			public void focusGained(FocusEvent e)
+			{
+				if (ACCOUNT_USERNAME.equals(String.valueOf(txtAccountLogin.getPassword())))
+				{
+					txtAccountLogin.setText("");
+					if (config.isStreamerMode())
+					{
+						txtAccountLogin.setEchoChar('*');
+					}
+					txtAccountLogin.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
+				}
+			}
+
+			@Override
+			public void focusLost(FocusEvent e)
+			{
+				if (txtAccountLogin.getPassword().length == 0)
+				{
+					txtAccountLogin.setForeground(ColorScheme.MEDIUM_GRAY_COLOR);
+					txtAccountLogin.setText(ACCOUNT_USERNAME);
+					txtAccountLogin.setEchoChar((char) 0);
+				}
+			}
+		});
+
+		add(txtAccountLogin, c);
+		c.gridy++;
+		c.insets = new Insets(0, 0, 15, 0);
+
+		JButton btnAddAccount = new JButton("Add Account");
+		btnAddAccount.setPreferredSize(PREFERRED_SIZE);
+		btnAddAccount.setBackground(ColorScheme.DARKER_GRAY_COLOR);
+		btnAddAccount.setMinimumSize(MINIMUM_SIZE);
+		btnAddAccount.addActionListener(e ->
+		{
+			String data = txtAccountLabel.getText() + ":" + String.valueOf(txtAccountLogin.getPassword());
```

Comment:
May want to prevent them from adding the default values and duplicate login names?